implementations/micrometer-registry-statsd/src/test/java/io/micrometer/statsd/internal/BufferingFluxTest.java

```diff
@@ -66,7 +67,7 @@ void bufferMultipleStrings() {
                 "fourteen bytes"
         );
 
-        Flux<String> buffered = BufferingFlux.create(source, "\n", 27, 1000);
+        Flux<String> buffered = BufferingFlux.create(source, "\n", 28, 1000);
```

Comment:
why would the `maxByteArraySize` need to change here? Could this be an off-by-one error somewhere in the implementation? Having to do such a small modification in a test is suspicious, and looks like breaking behavior: what is the implication on prod usages that would create a similar `BufferingFlux` with `27`?